src/FontStashSharp/DynamicSpriteFont.cs

```diff
@@ -16,7 +16,7 @@ namespace FontStashSharp
 {
 	public partial class DynamicSpriteFont: SpriteFontBase
 	{
-		internal readonly Int32Map<DynamicFontGlyph> Glyphs = new Int32Map<DynamicFontGlyph>();
+		internal readonly Int32Map<DynamicFontGlyph>[] Glyphs = new Int32Map<DynamicFontGlyph>[] { new Int32Map<DynamicFontGlyph>(), new Int32Map<DynamicFontGlyph>() };

```

Comment:
In order to make the code more understandable, can you introduce two fields(named say `GlyphsWithoutBitmap` and `Glyphs`) instead of array?

Reply:
Sure, I just tried to save me some time by using the array